src/components/Header.vue

```diff
@@ -35,18 +34,17 @@
 import { eventBus } from '../main.js';
 
 export default {
-    //props: ['shareLink'],
     data() {
         return {
-            shareLink: ''   // 0 = hiden
+            shareLink: ''   // 0 = textbox hiden
```

Comment:
redundant comment :), if you have another logic behind this, you should use boolean instead initial string

```
shareLink: false
```